website/markdown/posts/2020-02-26-version-1.3.0/post.mdx

```diff
@@ -0,0 +1,90 @@
+Thanks to [this work](https://github.com/tuist/tuist/pull/1006) that's not the case anymore.
+From Tuist 1.3.0,
+`tuist generate` generates the projects preserving that data.
+
+## Performance-related improvements
+
+The generation of Xcode project **needs to be fast.**
+Unfortunately, the performance of the generation logic is something that we have disregarded since the inception of the project, and now some users are hitting slow project generations.
+We decided it's time to tackle this so the first step that we took was adding a [helper utility](https://github.com/tuist/tuist/pull/957) that helps us benchmark the performance.
+Thanks to it,
+we'll be able to uncover performance bottlenecks and fail pull requests if we introduce regressions.
+
+Moreover, we made [some improvements](https://github.com/tuist/tuist/pull/980) in the logic that sorts the project groups and files getting an improvement of roughly 50%.
+
+## Project manifesto
+
+As the project grows,
+it's crucial for users, contributors, and maintainers to know what are our design pillars.
+For that reason,
+we updated our documentation to include a [manifesto](/docs/contribution/manifesto/) that contains a set of principles that are foundational to the design and development of Tuist.
+They help align users and contributors with the vision of the project.
+
+## Installation from Google Cloud Storage
+
+Tuist used to install new versions of Tuist by pulling the binaries from the GitHub releases.
+As soon as users reported, that caused some issues because GitHub's API returned limit errors.
+To mitigate this issue,
+we moved our artifacts to Google Cloud Storage.
+Moreover,
+we took the opportunity to add support for pulling binaries for incremental builds.
+For instance,
+if a developer wants to try out a specific commit,
+they can run `tuist local commit_sha` and Tuist will pull the binaries and make sure that we are running the right version.
+
+## Other fixes and improvements
+
+Tuist 1.3.0 also ships with minor fixes and improvements that are worth mentioning:
+
+- Added metal as a valid source extension - [Link](https://github.com/tuist/tuist/pull/1023)
+- Fixed false positive warnings during the linting process - [Link](https://github.com/tuist/tuist/pull/981)
+- Added support for target action paths without the `./` prefix - [Link](https://github.com/tuist/tuist/pull/997)
+- Ensured bcsymbolmap paths are consistently sorted
+- Updated XcodeProj to 7.8.0
+- Added "Base" to the list of known regions - [Link](https://github.com/tuist/tuist/pull/1021)
+
+## What’s next
+
+Our goal is to provide teams with a full-fledged tool that helps them deal with their projects at scale. The list of challenges that teams face when scaling up their projects is long:
+
+- Keep consistency across projects.
+- Ensure that projects are simple and that can be modularized easily.
+- Integrate the management of certificates into.
```

Comment:
I'd delete `into`

Reply:
The sentence was actually incomplete. I rephrased it to:

```
Seamless management of certificates and profiles.
```